configure

```diff
@@ -1022,12 +1042,23 @@ sub write_platform_macros {
       }
       if ($cross_compile) {
         print $PMG 'TAO_IDL = $(HOST_ACE)/bin/tao_idl', "\n";
+        if ($is_windows) {
+          print $PMG "HOST_EXE_EXT = .exe\n";
+        }
         print $PMG 'TAO_IDL_DEP = $(TAO_IDL)$(HOST_EXE_EXT)', "\n";
-        print $PMG 'TAO_IDL_PREPROCESSOR = ', $opts{'compiler'}, "\n";
-        print $PMG 'TAO_IDLFLAGS += -g $(HOST_ACE)/bin/ace_gperf', "\n";
+        if ($has_host_compiler) {
+          print $PMG 'TAO_IDL_PREPROCESSOR = ', $opts{'compiler'}, "\n";
+        }
+        print $PMG 'TAO_IDLFLAGS += -g $(HOST_ACE)/bin/ace_gperf$(HOST_EXE_EXT)', "\n";
```

Comment:
```suggestion
        print $PMG 'TAO_IDLFLAGS += -g $(HOST_ACE)/bin/ace_gperf', "\n";
```
Should be OK without it since this is a command line not a file name.